src/DataFixtures/SkillsFixtures.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Skills;
+use Doctrine\Persistence\ObjectManager;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+
+class SkillsFixtures extends Fixture
+{
+    public const SKILLS = [
+        [
+            'name' => 'Achat',
+            'category' => 'Commerce et logistique',
```

Comment:
If it's one of the possible skill categories as defined in the CATEGORIES constant, you should use your constant here instead of duplicating the value